submissions/sprint_b/test_helpers.py

```diff
@@ -0,0 +1,79 @@
+from sprint_b.helpers import FileManager, Vocabulary, EpithetGenerator
```

Comment:
Imports should be grouped in the following order:
Standard library imports.
Related third party imports.
Local application/library specific imports.
You should put a blank line between each group of imports.

https://www.python.org/dev/peps/pep-0008/#imports